lib/topologies/shared.js

```diff
@@ -217,7 +217,8 @@ function Timeout(fn, time) {
 
   this.start = function () {
     if (!this.isRunning()) {
-      timer = setTimeout(fn, time);
+      // The artificial _called is set here for compatibility with node.js 0.10.x/0.12.x versions
+      timer = setTimeout(function() { fn(); if (timer !== false && timer._called === undefined) { timer._called = true; } }, time);
```

Comment:
Hi @ilguzin,

Can you please incorporate the following changes:

1. replace `timer !== false` with just `timer`
2. Break this all out into multiple lines.

Ideally, it would be something like this?

```js
timer = setTimeout(function() {
  fn();
  if (timer && timer._called === undefined) {
    timer._called = true;
  }
}, time);
```